internal/url/url.go

```diff
@@ -327,3 +327,20 @@ func SSOAuthAction(action Action) Action {
 	}
 	return action
 }
+
+const AWS_FEDERATED_URL_FORMAT = "https://%s.signin.%s/federation"
+
+// AWSFederatedUrl generates the region/partition specific URL for the AWS
+// Federated endpoint for IAM Identity Center
+// https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_providers_enable-console-custom-url.html
+func AWSFederatedUrl(region string) string {
+	if strings.HasPrefix(region, "cn-") {
+		// china
+		return fmt.Sprintf(AWS_FEDERATED_URL_FORMAT, region, "amazonaws.com.cn")
```

Comment:
```suggestion
		return fmt.Sprintf(AWS_FEDERATED_URL_FORMAT, region, "amazonaws.cn")
```
Command works after making this change. However, in the browser window i'm still seeing this
<img width="1128" alt="Screenshot 2023-10-19 at 16 11 14" src="https://github.com/synfinatic/aws-sso-cli/assets/3320547/6dd111df-fbde-443e-8772-35d430f98fbb">